### PR TITLE
Implemented adding areas to centers

### PR DESCRIPTION
**Pull request by ccapiferri**

### CODE REVIEW

File: climate-monitoring-server/src/main/java/climatemonitoring/Skeleton.java [hunk 1]
```diff
@@ -500,6 +500,25 @@ public void run() {
...
+						}
+
+						catch (DatabaseRequestException e) {
+	
+							m_out.writeObject(false);
+							m_out.writeObject(e);
+						}
+					}
+
 					case MONITORS: {
```
Comment (adellafrattina) [inline climate-monitoring-server/src/main/java/climatemonitoring/Skeleton.java]: You are missing a `break`

**Review by adellafrattina — CHANGES_REQUESTED** [review]

File: climate-monitoring-server/src/main/java/climatemonitoring/Skeleton.java (hunk 1 above)
Comment (adellafrattina) [reply climate-monitoring-server/src/main/java/climatemonitoring/Skeleton.java]: 79.50.205.224

**Comment by adellafrattina** [comment]
Could you add a `getParameter(geoname_id)` method to return parameters based only on the specified area. The center will be selected based on the most recent parameter. Remember to create the correct RequestType

**Comment by ccapiferri** [comment]
No

**Comment by adellafrattina** [comment]
kys

**Review by adellafrattina — APPROVED** [review]
LGTM!

**Pull request merged**